src/test/scala/uk/gov/hmrc/govukfrontend/views/implicits/RichRadiosSpec.scala

```diff
@@ -63,6 +64,28 @@ class RichRadiosSpec extends AnyWordSpec with Matchers with MessagesHelpers with
       )
     }
 
+    "divider should not be marked as checked when field value is set to None" in {
+      val field: Field = Field(
+        form = form,
+        name = "user-name",
+        constraints = Nil,
+        format = None,
+        errors = Seq(),
+        value = None
+      )
+
+      val radioItem1       = RadioItem(content = Text("This is good"), value = Some("good"), checked = true)
+      val radioItemDivider = RadioItem(divider = Some("Divider"), checked = false)
+      val radioItem2       = RadioItem(content = Text("This is THE WORST"), value = Some("worst"), checked = false)
+
+      val radios = Radios(items = Seq(radioItem1, radioItemDivider, radioItem2)).withFormField(field)
+      radios.items shouldEqual Seq(
+        radioItem1,
+        radioItemDivider,
+        radioItem2
+      )
+    }
+
```

Comment:
what do you think about slightly trimming down the test? (haven't checked if this worked)

```suggestion
    "divider should not be marked as checked when field value is set to None" in {
      val emptyFormField = Field(
        form = Form(),
        name = "field",
        constraints = Nil,
        format = None,
        errors = Seq(),
        value = None
      )

      val dividerFromRadiosUpdatedWithEmptyFormField = Radios(
          items = Seq(
             RadioItem(divider = Some("or"))
          )
      ).withFormField(emptyFormField).items.head
      
      dividerFromRadiosUpdatedWithEmptyFormField shouldEqual RadioItem(
          divider = Some("or"),
          checked = false
      )
    
    }

```

Reply:
@oscarduignan  Looks much better, thanks for the suggestion, i have updated the code.